src/main/java/org/folio/rs/domain/dto/DomainEventType.java

```diff
@@ -0,0 +1,5 @@
+package org.folio.rs.domain.dto;
+
+public enum DomainEventType {
```

Comment:
There is another type of event. If you don't declare it here, there will be an exception